package.json

```diff
@@ -12,15 +12,16 @@
     "fetch-sponsors": "node scripts/fetch-sponsors.js"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-alpha.72",
-    "@docusaurus/plugin-google-analytics": "^2.0.0-alpha.66",
-    "@docusaurus/preset-classic": "^2.0.0-alpha.72",
+    "@docusaurus/core": "2.0.0-alpha.75",
+    "@docusaurus/plugin-google-analytics": "2.0.0-alpha.75",
```

Comment:
Thanks @slorber for your PR! just one small thing to check before we land it.

@brentvatne could you check if this new version GA plugin still working the analytics?

Reply:
Thanks

Do you expect me to do anything?

Reply:
nope, all looks good to me @slorber, thanks again for your awesome work on docusarus.

Reply:
@eriveltonelias - i can add you to the google analytics account, dm me on discord to let me know what email to add :)